optimum/exporters/openvino/convert.py

```diff
@@ -659,7 +660,11 @@ def export_from_model(
         files_subpaths = ["openvino_" + model_name + ".xml" for model_name in models_and_export_configs.keys()]
     elif library_name != "diffusers":
         if is_transformers_version(">=", "4.44.99"):
-            misplaced_generation_parameters = model.config._get_non_default_generation_parameters()
+            # some model configs may have issues with loading without parameters initialization
+            try:
+                misplaced_generation_parameters = model.config._get_non_default_generation_parameters()
+            except Exception:
+                misplaced_generation_parameters = {}
```

Comment:
There may be some valid useful exceptions hidden this way. Could we come up with a more selective way to solve the current problem?

Reply:
in general, we do not know original reason why it may fail. In my particular example, model config class is not designed to be loaded without that's that used inside get_non_default_generation_parameters

https://huggingface.co/OpenGVLab/InternVL2-1B/blob/main/configuration_internvl_chat.py#L45

here if not language_config provided, it initialized as empty dict, but next line trying to get "architectures" key without checking that this key provided https://huggingface.co/OpenGVLab/InternVL2-1B/blob/main/configuration_internvl_chat.py#L49

I pointed on this error to model authors, but they just ignoring me and suggest to downgrade transformers
https://huggingface.co/OpenGVLab/InternVL2-1B/discussions/7

looking on this issue, keeping me thinking that other models may fail with different error and reason and I do not think that I'll have opportunity to update that in future, that is why I think it is better to keep it generic as config still may be loaded and can not be saved due to optional checks that become obligated

Reply:
only KeyError catchednow